src/manager/network-manager.c

```diff
@@ -2713,6 +2713,7 @@ int manager_set_ipv4(const IfNameIndex *ifidx, const int dhcp, const IPAddress *
         _auto_cleanup_ char *network = NULL, *gw = NULL, *addr = NULL;
         _cleanup_(key_file_freep) KeyFile *key_file = NULL;
         _cleanup_(section_freep) Section *section = NULL;
+        DHCPClient mode = DHCP_CLIENT_NO;
         int r;
```

Comment:
By default it should be ```_DHCP_CLIENT_INVALID```

Reply:
> By default it should be `_DHCP_CLIENT_INVALID`

Okay will change it but in the case of _DHCP_CLIENT_INVALID also need to set whatever passed from CLI args.